manifests/plugin/dns_ovh.pp

```diff
@@ -0,0 +1,59 @@
+# == Class: letsencrypt::plugin::dns_ovh
```

Comment:
Could you format this using puppet-strings style?

Reply:
Yes, it's done [here](https://github.com/aellert/puppet-letsencrypt/commit/1175fcc8073f2affb27a124367e017592c0aba83)
Should I do another PR ?

Reply:
It's fine here. A separate PR to convert the rest of the module would be appreciated though :)

Reply:
Ok, next week I will probably have some time to do it !